README.md

```diff
@@ -6,22 +6,33 @@ Create baseline images and test for CSS regression during standard Ember tests u
 
 `ember install ember-cli-visual-acceptance`
 ### Configuration
-You can modify the save directory in `ember-cli-build.js` by including
-```
+* You can modify the save directory in `ember-cli-build.js` by including
+```javascript
 visualAcceptanceOptions: {
       imageDirectory: 'visual-acceptance'
     }
 ``` 
+* You can specify browsers to target by adding an object inside the `targetBrowsers` array in the `visualAcceptanceOptions` of `ember-cli-build.js`
+```javascript
+visualAcceptanceOptions: {
+      targetBrowsers: [{
+        bit: undefined,
```

Comment:
Would it make sense to chop this down to just `browser` and `os` as mandatory?  I'd probably use `>=` for the `osversion` and `version` and I'm not sure what `bit` and `mobile` are supposed to control